packages/controller/lib/setup.js

```diff
@@ -579,12 +579,28 @@ async function processCommand(command, args, params, callback) {
                             // And try to install each of them
                             for (const instance of initialInstances) {
                                 try {
-                                    const path = require.resolve(`${tools.appName}.${instance}`);
-                                    if (path) {
-                                        await install.createInstance(instance, {
-                                            enabled: true,
-                                            ignoreIfExists: true
+                                    let otherInstanceExists = false;
```

Comment:
I would move that into "if path" because we only need to check if adater is installed ... or!?

Reply:
or checks vice versa both ok